engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -259,6 +262,7 @@ SELECT
     `network_offerings`.`for_vpc` AS `for_vpc`,
     `network_offerings`.`for_tungsten` AS `for_tungsten`,
     `network_offerings`.`for_nsx` AS `for_nsx`,
+    `network_offerings`.`mode` AS `mode`,
```

Comment:
I think we can call it nsx_mode as other types will have this field null by default